mesonbuild/mesonlib.py

```diff
@@ -1,7 +1,7 @@
 # SPDX-license-identifier: Apache-2.0
 # Copyright 2012-2021 The Meson development team
 # Copyright © 2021-2023 Intel Corporation
-
+#
 # pylint: skip-file
```

Comment:
... which leads to all these changes, which seem a bit pointless as they are solely to help isort apparently be substandard about injecting imports? My gut feeling here is that this is a bug in isort.